src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/AppleUninstallCommand.cs

```diff
@@ -21,7 +22,7 @@ internal class AppleUninstallCommand : AppleAppCommand<AppleUninstallCommandArgu
         private const string CommandHelp = "Uninstalls a given iOS/tvOS/watchOS/MacCatalyst application bundle from a target device/simulator";
 
         protected override AppleUninstallCommandArguments AppleAppArguments { get; } = new();
-        protected override string CommandUsage { get; } = "apple uninstall [OPTIONS]";
+        protected override string CommandUsage { get; } = "apple uninstall --app=... --output-directory=... --targets=... [OPTIONS] [-- [RUNTIME ARGUMENTS]]";
```

Comment:
Do we really need `output-directory` for the `uninstall` command?

Reply:
There's various log files (device logs, mlaunch logs) that we capture and put in this directory (for Helix, this all is uploaded then with the results).